test/jest/victory-tooltip/victory-tooltip.test.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Client tests
+ */
+/* global console */
+import React from "react";
+import { VictoryTooltip } from "victory-tooltip";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+describe("components/victory-tooltip", () => {
+  const baseProps = {
+    x: 0,
+    y: 0,
+    datum: { some: "object" },
+    index: 3,
+    active: true,
+    text: "such text, wow"
+  };
+  beforeAll(() => {
+    jest.spyOn(console, "warn").mockImplementation(() => {});
+    jest.spyOn(console, "error").mockImplementation(() => {});
+  });
+
+  it("renders nothing when not active", () => {
+    render(<VictoryTooltip {...baseProps} active={false} />);
+    const output = screen.findByText(baseProps.text);
+    expect(output).toBeDefined();
+  });
+
+  it("has expected text", () => {
+    render(<VictoryTooltip {...baseProps} />);
+    const output = screen.findByText(baseProps.text);
+    expect(output).toBeDefined();
+  });
+
+  it("renders a flyout and a label", () => {
+    const { container } = render(<VictoryTooltip {...baseProps} />);
+    const label = container.querySelector("text");
+    const flyout = container.querySelector("path");
```

Comment:
Nitpick: We can use the `flyoutComponent` and `labelComponent` props here to pass in components with a test id and use `findByTestId`. That way we can be more specific and confirm that these component we are finding are actually being rendered as the flyout and label, and not some other SVG child.

Reply:
If you go for that option, we will need to add the `getSafeUserProps` function to `Flyout` like I did for `VictoryLabel` in my last PR.

Reply:
Done!